### ISSUE TITLE
error when test code of box_generator using command python sample.py --is_training 0

### ISSUE THREAD

**Issue by fyw1999** [jamesli1618/Obj-GAN#29]
I want to test box_generator,so i execute the command "python sample.py --is_training 0". But i get the following error:
File "/home/fyw/code/objGan/box_generation/seq2seq/models/DecoderRNN.py", line 430, in sample_bivariate_normal
    x = np.random.multivariate_normal(mean, cov, 1)
  File "mtrand.pyx", line 4114, in numpy.random.mtrand.RandomState.multivariate_normal
TypeError: ufunc 'add' output (typecode 'O') could not be coerced to provided output parameter (typecode 'd') according to the casting rule ''same_kind''

How can i solve this error? Please help me!

**Comment by GuoxingY** [comment]
Have you solved this error? I meet the same error now!

**Comment by 917596622** [comment]
Yes, I have solved this error. But it's been so long that I've forgotten how I solved it. I can tell you the environment configuration where the code can run successfully. I used GTX 1080Ti+python3.7+pytorch1.3.0+torchvision0.4.1+cudatoolkit10.0.130. This error should be caused by environment. I have used many types of GPUS and I found in this configuration the code can run.

**Comment by fyw1999** [comment]
> Have you solved this error? I meet the same error now!

Yes, I have solved this error. But it's been so long that I've forgotten how I solved it. I can tell you the environment configuration where the code can run successfully. I used GTX 1080Ti+python3.7+pytorch1.3.0+torchvision0.4.1+cudatoolkit10.0.130. This error should be caused by environment. I have used many types of GPUS and I found in this configuration the code can run.

**Comment by GuoxingY** [comment]
> > Have you solved this error? I meet the same error now!
> 
> Yes, I have solved this error. But it's been so long that I've forgotten how I solved it. I can tell you the environment configuration where the code can run successfully. I used GTX 1080Ti+python3.7+pytorch1.3.0+torchvision0.4.1+cudatoolkit10.0.130. This error should be caused by environment. I have used many types of GPUS and I found in this configuration the code can run.

Thanks so much. I wonder this error ocuur because the types of GPUs or the environment ? I use python 3.6.3 + pytorch1.7.1 + torchvision 0.8.2 now. Maybe can solve this error by installing lower torch version ?

**Comment by 917596622** [comment]
That's probably true. I remember encountering this error when I used 2080Ti. Finally, I successfully executed the code on 1080Ti. You can attempt to install lower torch version firstly.

**Comment by fyw1999** [comment]
> > > Have you solved this error? I meet the same error now!
> > 
> > 
> > Yes, I have solved this error. But it's been so long that I've forgotten how I solved it. I can tell you the environment configuration where the code can run successfully. I used GTX 1080Ti+python3.7+pytorch1.3.0+torchvision0.4.1+cudatoolkit10.0.130. This error should be caused by environment. I have used many types of GPUS and I found in this configuration the code can run.
> 
> Thanks so much. I wonder this error ocuur because the types of GPUs or the environment ? I use python 3.6.3 + pytorch1.7.1 + torchvision 0.8.2 now. Maybe can solve this error by installing lower torch version ?

That's probably true. I remember encountering this error when I used 2080Ti. Finally, I successfully executed the code on 1080Ti. You can attempt to install lower torch version firstly.

**Comment by GuoxingY** [comment]
Thanks for your help again!

**Comment by TimStefany** [comment]
Hey I am stuck with the same error currently... 
I already tried different versions of torch and other dependencies, so I wanted to ask if someone who was able to run the box_generator could provide version requirements?
Thanks in advance!

**Comment by 917596622** [comment]
> Hey I am stuck with the same error currently...
> I already tried different versions of torch and other dependencies, so I wanted to ask if someone who was able to run the box_generator could provide version requirements?
> Thanks in advance!

I used GTX 1080Ti+python3.7+pytorch1.3.0+torchvision0.4.1+cudatoolkit10.0.130.

**Comment by fyw1999** [comment]
> Hey I am stuck with the same error currently...
> I already tried different versions of torch and other dependencies, so I wanted to ask if someone who was able to run the box_generator could provide version requirements?
> Thanks in advance!

I used GTX 1080Ti+python3.7+pytorch1.3.0+torchvision0.4.1+cudatoolkit10.0.130.  Under this version requirements, the code can run successfully.

**Comment by GuoxingY** [comment]
python3.7+pytorch1.3.0+torchvision0.4.1+cudatoolkit10.0.130. I can successfully run the code with this environment. Note that you should install the torch torchvision and cudatoolkit with the command "conda install ....", I failed when I used "pip install ..."

**Comment by TimStefany** [comment]
Thanks for the quick reply! I already tried with pytorch1.3.0+torchvision0.4.1+cudatoolkit10.0.130 but I am then running into problems that might be caused by the numpy and scipy version. Would you be able to tell me your version for the other dependencies as well?
I am trying this on colab so if I would like to avoid installing conda.
